MetaMask/metamask-extension
# fix: getMinimumGasTotalInHexWei error when gasEstimateType === "fee_market"

## **Description**

See Issue https://github.com/MetaMask/metamask-extension/issues/24263 for investigation details

This will be a hotfix targeting v11.14.3

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/24263

## **Manual testing steps**

One way to repro is by adding the Scroll Sepholia Testnet (see: https://docs.scroll.io/en/user-guide/setup/), then using our test dapp > Send Eth > Send

For other repro methods, please refer to the related issue ticket

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
